_posts/2023-09-14-load-up-on-sveltekit.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Load Up on SvelteKit"
+summary: "A primer talk on Svelte and SvelteKit with a dive into data loading with practical examples"
+date: 2023-09-14T19:15:00
+meetup: https://www.meetup.com/async-web-tech-meetup/events/295822326/
+speakers:
+  - name: Scott Spence
+    link: https://scottspence.com/
+sponsors:
+image:
+  url: /img/talks/sveltekit.jpg
+  title: SvelteKit
+tags:
+  - framework
+  - svelte
+  - sveltekit
+  - data
+venue:
+  name: Runway East
+  link: https://runwayea.st/locations/brighton?utm_source=external&utm_medium=event&utm_campaign=sponsorship
+  location: https://goo.gl/maps/WGe2p9D7Wup3LdNt8
+  address: York And Elder Works, 50 New England St, Brighton BN1 4AW
+  latlong: 50.8325788,-0.1420808
+layout: event.html
+collection: events
+---
+
+A Svelte and SvelteKit primer talk with a dive into data loading with practical examples.
```

Comment:
```suggestion
On the 14th of September (the second Thursday of the month) Scott Spence will be giving a Svelte and SvelteKit primer talk with a dive into data loading with practical examples.
```

Think it worth mentioning the change of date, and full name in the body of the event.
If you're cool with this, I'll update the Meetup similarly and add the speaker cards.